CHANGES.rst

```diff
@@ -132,6 +132,10 @@ Release History
   `#671 <https://github.com/nengo/nengo/pull/671>`_)
 - Fix for SPA actions that route to a module with multiple inputs.
   (`#714 <https://github.com/nengo/nengo/pull/714>`_)
+- Corrected the `'rmses'` values in `BuiltConnection.solver_info` when using
+  `NNls` and `Nnl2sL2` solvers. Previous values were for the Gram matrix, and
```

Comment:
If I were to cut it off after the first sentence, then there would be nothing about regularization. So I merged the third sentence into the first sentence and removed the second. Also added double backticks and linked this PR. That said, I wasn't really trying to summarize git; the second sentence was there because there is a separate but connected user-facing change (i.e. `solver_info` now has another key). But I can buy the argument that this is not actually user-facing since nobody yet uses those solvers.